src/components/Feed/Feed.tsx

```diff
@@ -36,7 +38,7 @@ const Feed = ({ unreadArticles, updateUserSentiment, storeArticle }: FeedProps):
       <Header />
       <div className="articles-container">
         <section className="articles-display">
-          { articleCards.length ? articleCards :
+          { articleCards.length > 0 ? articleCards :
```

Comment:
Curious why we needed to add the greater than 0. Was it not evaluating as truthy/falsy?